src/Modules/GraphEngine.DynamicCluster/Trinity.DynamicCluster/Communication/DynamicClusterCommModule.cs

```diff
@@ -178,15 +178,15 @@ public override void GetChunksHandler(GetChunksResponseWriter response)
 
         public override void AnnounceMasterHandler(StorageInformationReader request, ErrnoResponseWriter response)
         {
-            (m_memorycloud as DynamicMemoryCloud).m_cloudidx.SetMaster(request.partition, request.id);
+            (m_memorycloud as DynamicMemoryCloud)?.m_cloudidx.SetMaster(request.partition, request.id);
```

Comment:
nit: should not silently return E_OK when m_memorycloud resolution fails.

Reply:
Restored to the original statement.